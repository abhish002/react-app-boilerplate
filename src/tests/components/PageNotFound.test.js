import React from 'react';
import { shallow } from 'enzyme';
import  PageNotFound from '../../components/PageNotFound';

test('should render page not found component', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});