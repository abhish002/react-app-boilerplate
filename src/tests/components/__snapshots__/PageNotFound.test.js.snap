// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page not found component 1`] = `
<div>
  404! Page Not Found! 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
